testsuite/scripts/src/test/java/org/wildfly/scripts/test/StandaloneScriptTestCase.java

```diff
@@ -86,6 +89,28 @@ public static Collection<Object> data() {
         return result;
     }
 
+    @Test
+    public void testBatchScriptJavaOptsToEscape() throws Exception {
+        Assume.assumeTrue(!TestSuiteEnvironment.isWindows() && Shell.BASH.isSupported());
```

Comment:
The assume condition needs to be fixed; it should be: `Assume.assumeTrue(TestSuiteEnvironment.isWindows())`

Although the test should pass on Linux, we don't need to run it there, indeed, looking at the method name, that was not your original intention.

Reply:
Oh sure, thanks @yersan, I don't know how it got there. :-)

Reply:
:) thanks, @lvydra